src/Dashboard/ApiControllers/DiagnosticsController.cs

```diff
@@ -1,117 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Web.Http;
```

Comment:
I could, but this would also mean backporting issues over to the 2.x branch. This may become tricky to do in the current branch because of how the code dependency is setup, which will become more challenging as the code diverges.